PurchasesCoreSwift/IntroEligibilityCalculator.swift

```diff
@@ -39,26 +40,34 @@ import StoreKit
         }
         do {
             let receipt = try receiptParser.parse(from: receiptData)
-            let purchasedProductIdsWithIntroOffersOrFreeTrials = receipt.purchasedIntroOfferOrFreeTrialProductIdentifiers()
-            
-            let allProductIdentifiers = candidateProductIdentifiers.union(purchasedProductIdsWithIntroOffersOrFreeTrials)
-            
+            // swiftlint:disable:next identifier_name
```

Comment:
Since we really do want these verbose names, maybe we can configure `identifier_name` with:
```
variable_name:
  max_length:
    warning: 60
    error: 80
```

Thoughts?

Reply:
yes, much better, I don't have anything against names like these